src/components/MessageHTMLBody.vue

```diff
@@ -134,7 +134,7 @@ export default {
 		async onShowBlockedContentForDomain() {
 			this.displayIframe()
 			// TODO: there might be more than one @ in an email address
-			await trustSender(this.getDomain(), 'domain', true)
+			await trustSender(this.domain(), 'domain', true)
```

Comment:
does this work? :thinking:

Reply:
yes, no error is shown and the action is saved correctly. _getDomain_ was removed here: https://github.com/nextcloud/mail/commit/29dddbfc38b63226f0e3af5fb5612ae3782098aa and _domain_ was added but not replaced in this method.

Reply:
https://vuejs.org/v2/guide/computed.html#Computed-Caching-vs-Methods while it is allowed and correct to use computed properties as method I would drop the `()` like we do in other code